Streetcode/Streetcode.WebApi/Controllers/Newss/NewsController.cs

```diff
@@ -22,6 +23,12 @@ public async Task<IActionResult> GetById(int id)
             return HandleResult(await Mediator.Send(new GetNewsByIdQuery(id)));
         }
 
+        [HttpGet]
+        public async Task<IActionResult> SortedByDateTime()
+        {
+            return HandleResult(await Mediator.Send(new SortedByDateTimeQuery()));
```

Comment:
Do you mind naming this method something like SortedNewsByDateTime? 'cause I needed some time to process what do you need to sort specifically.